package.json

```diff
@@ -41,7 +41,7 @@
     "react-card-flip": "^1.0.11",
     "react-dom": "^17.0.2",
     "react-gtm-module": "^2.0.11",
-    "react-hook-form": "^6.13.1",
+    "react-hook-form": "^7.6.6",
```

Comment:
Updating for access to some new features and to keep up to date with the latest examples.